cl.conf

```diff
@@ -13,6 +13,15 @@ systemd:
 
         [Install]
         WantedBy=sockets.target
+    - name: var-amobee.mount
+      enable: true
+      contents: |
+        [Mount]
+        What=/dev/disk/by-id/ata-VBOX_HARDDISK_VBb121c145-7f71a112-part1
```

Comment:
is `ata-VBOX_HARDDISK_VBb121c145-7f71a112-part1` arbitrary or based on something?

Reply:
It is based on the UUID when the storage medium is attached. it is ata-VBOX_HARDDISK_VB<first UUID octet>-<last 8 uuid characters starting from the end grabbed in pairs (i.e. last eight chars = 12A1717F -> in pairs 12 A1 71 7F -> grouped from end 7f71a112 )>. It is the only predictable disk location I could find.

Reply:
Hm... we should be able to load this disk directly by UUID. Do you see anything in /dev/disk/by-uuid ? I would expect that our UUID shows up there.

Reply:
When you set the disk UUID on attach it is the UUID virtualbox uses for that disk. It doesn't seem to correspond to the disk UUID in the vm, the `ata-VBOX_HARDDISK_VBb121c145-7f71a112-part1` id was the only predictable disk reference I could find.